docs/src/components/productX/XHero.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from 'react';
+import { XGrid } from '@material-ui/x-grid';
+import { useDemoData } from '@material-ui/x-grid-data-generator';
+import Box from '@mui/material/Box';
+import Divider from '@mui/material/Divider';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import { DateRange } from '@mui/lab/DateRangePicker';
+import StaticDateRangePicker from '@mui/lab/StaticDateRangePicker';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import GradientText from 'docs/src/components/typography/GradientText';
+import GetStartedButtons from 'docs/src/components/home/GetStartedButtons';
+import HeroContainer from 'docs/src/layouts/HeroContainer';
+import IconImage from 'docs/src/components/icon/IconImage';
+import FolderTreeView from 'docs/src/components/showcase/FolderTreeView';
+
+const startDate = new Date();
+startDate.setDate(10);
+const endDate = new Date();
+endDate.setDate(endDate.getDate() + 28);
+
+export default function XHero() {
+  const { loading, data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 10000,
+    maxColumns: 40,
+    editable: true,
+  });
+  const [value, setValue] = React.useState<DateRange<Date>>([startDate, endDate]);
+  return (
+    <HeroContainer
+      left={
+        <Box sx={{ textAlign: { xs: 'center', md: 'left' } }}>
+          <Typography
+            fontWeight="bold"
+            variant="body2"
+            color={(theme) => (theme.palette.mode === 'dark' ? 'primary.400' : 'primary.600')}
+            sx={{
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: { xs: 'center', md: 'flex-start' },
+              '& > *': { mr: 1, width: 28, height: 28 },
+            }}
+          >
+            <IconImage name="product-advanced" /> MUI X
+          </Typography>
+          <Typography variant="h1" sx={{ my: 2, maxWidth: 500 }}>
+            Performant
+            <br />
+            <GradientText>advanced</GradientText>
+            <br /> components
+          </Typography>
+          <Typography color="text.secondary" sx={{ mb: 3, maxWidth: 500 }}>
+            Build complex and data-rich applications using a growing list of advanced components.
+            We&apos;re kicking it off with the most powerful Data Grid on the market.
+          </Typography>
+          <GetStartedButtons
+            installation="npm install @mui/x-data-grid"
+            sx={{ justifyContent: { xs: 'center', md: 'flex-start' } }}
+          />
```

Comment:
Something is broken with the keyboard navigation. Here is what I get when I tab a couple of times.

<img width="546" alt="Screenshot 2021-09-05 at 13 59 04" src="https://user-images.githubusercontent.com/3165635/132125960-0e2622cd-20a2-49c2-99ef-05681f116c80.png">

I can reproduce on https://next--material-ui.netlify.app/components/buttons/ too. My first guess was that it's coming from #27600, but no

Reply:
https://github.com/mui-org/material-ui/issues/28181

Michal is working on it.